website/content/api-docs/system/replication/index.mdx

```diff
@@ -197,3 +199,69 @@ performance secondary and DR primary node, it will look something like:
   }
 }
 ```
+
+## Check Merkle Tree Corruption
```

Comment:
I think other examples of endpoint docs use the endpoint path as the header. We should probably do the same here for consistency.

```suggestion
## `/sys/replication/merkle-check`
```